spinalcordtoolbox/vertebrae/core.py

```diff
@@ -473,13 +541,16 @@ def label_segmentation(fname_seg, list_disc_z, list_disc_value, verbose=1):
     """
 
     # open segmentation
+    # sct.run('sct_resample -i %s -mm 1x1x1 -x nn -o segmentation_straight.nii'%(fname_seg))
+    # fname_seg = 'segmentation_straight.nii'
     seg = Image(fname_seg)
     init_orientation = seg.orientation
     seg.change_orientation("RPI")
 
     dim = seg.dim
     ny = dim[1]
     nz = dim[2]
+    print(list_disc_z)
```

Comment:
Nitpick: Is this print statement for testing or for the user? (Might need to be cleaned up.)

Reply:
Testing. I will remove it !

Reply:
done in cb268fd